apps/heureka/src/components/Services/ServicesList/ServicesList.tsx

```diff
@@ -70,7 +69,7 @@ export const ServicesList = ({
           <DataGridHeadCell>Service details</DataGridHeadCell>
           <DataGridHeadCell>No. Components</DataGridHeadCell>
           <DataGridHeadCell>Service owners</DataGridHeadCell>
-          <DataGridHeadCell></DataGridHeadCell>
+          <DataGridHeadCell>Actions</DataGridHeadCell>
```

Comment:
Do we really need the this header? We normally don't set header on the actions column. See supernova and heureka wireframes.

![Screenshot 2025-03-28 at 11 14 40](https://github.com/user-attachments/assets/98bbea28-f9dd-44dd-b7e6-1dac71895df1)